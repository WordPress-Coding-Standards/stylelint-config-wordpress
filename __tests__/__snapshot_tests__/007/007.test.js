'use strict';

const stylelint = require( 'stylelint' ),
	systemTestUtils = require( '../systemTestUtils.js' );

it( '007', () => {
	return stylelint.lint({
		files: [systemTestUtils.caseStylesheetGlob( '007' )],
		configFile: systemTestUtils.caseConfig( '007' ),
	}).then( ({ results }) => {
		expect( systemTestUtils.prepResults( results ) ).toMatchSnapshot();
	});
});
