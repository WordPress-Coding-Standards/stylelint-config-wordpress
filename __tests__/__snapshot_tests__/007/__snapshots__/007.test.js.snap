// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`007 1`] = `
Array [
  Object {
    "deprecations": Array [],
    "errored": true,
    "ignored": undefined,
    "invalidOptionWarnings": Array [],
    "parseErrors": Array [],
    "source": "007/stylesheet.scss",
    "warnings": Array [
      Object {
        "column": 18,
        "line": 8,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 46,
        "line": 8,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 23,
        "line": 8,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 35,
        "line": 8,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 22,
        "line": 18,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 33,
        "line": 18,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 25,
        "line": 38,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 55,
        "line": 38,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 34,
        "line": 39,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 50,
        "line": 39,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 42,
        "line": 40,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 78,
        "line": 40,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 51,
        "line": 40,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 72,
        "line": 40,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 33,
        "line": 55,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 54,
        "line": 55,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 37,
        "line": 56,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 58,
        "line": 56,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 28,
        "line": 58,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace after \\"(\\" (function-parentheses-space-inside)",
      },
      Object {
        "column": 58,
        "line": 58,
        "rule": "function-parentheses-space-inside",
        "severity": "error",
        "text": "Unexpected whitespace before \\")\\" (function-parentheses-space-inside)",
      },
    ],
  },
]
`;
